Pipfile

```diff
@@ -4,36 +4,22 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-asgiref = "==3.4.1"
-click = "==8.0.1"
-fastapi = "==0.66.0"
-h11 = "==0.12.0"
-httptools = "==0.2.0"
-mmh3 = "==3.0.0"
-pottery = "==1.3.1"
-pydantic = "==1.8.2"
-python-dotenv = "==0.18.0"
-python-twitter = "==3.5"
-redis = "==3.5.3"
-starlette = "==0.14.2"
-typing-extensions = "==3.10.0.0"
-lunr = "==0.6.0"
-urllib3 = "==1.26.6"
-yarl = "==1.6.3"
-uvicorn = "==0.15.0"
-python-decouple = "==3.4"
-PyYAML = "==5.4.1"
-Pyrogram = "==1.2.9"
```

Comment:
This would remove a ton of dependencies.

Reply:
Now using the same Pipfile/requirements as `dev`.